pages/_app.js

```diff
@@ -52,6 +53,7 @@ export default function App({ Component, pageProps }) {
           setQuery={setQuery}
         />
       </Main>
+      <Layout />
```

Comment:
I wouldn't use components inside `_app.js` this way, because everything can go inside the `index.js`.

Reply:
I think this layout component is ok here, if its wrapped around the Component.